ui/pages/settings/advanced-tab/advanced-tab.container.js

```diff
@@ -13,6 +13,7 @@ import {
   setFeatureFlag,
   setShowFiatConversionOnTestnetsPreference,
   setShowTestNetworks,
+  setShowExtensionInFullSizeView,
```

Comment:
[super nit] placement breaks alphabetical prop ordering here and other places above